platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/MerkleDbTableConfig.java

```diff
@@ -148,17 +149,20 @@ public MerkleDbTableConfig() {
      *      Hash version
      * @param hashType
      *      Hash type
+     * @param merkleDbConfig platform config for MerkleDbDataSource
      */
-    public MerkleDbTableConfig(final short hashVersion, final DigestType hashType) {
+    public MerkleDbTableConfig(
+            final short hashVersion, final DigestType hashType, final @NonNull MerkleDbConfig merkleDbConfig) {
```

Comment:
How difficult would it be to add `maxNumberOfKeys` and `hashesRamToDiskThreshold` to the list of args? This would eliminate need to store the configuration in this object

Reply:
Done, see this commit (waiting for checks to pass):https://github.com/hashgraph/hedera-services/pull/15562/commits/568b04a84952d64f8f847c9e123d37fe3ff687ba